### DISCUSSION TITLE
Update sabnzbd

### DISCUSSION THREAD

**Discussion by ApriliaEdd** [tteck/Proxmox#1199] [Q&A] [answered]
how do you update the service in the sabnzbd script?  running the script again in the lxc console updates the os but sabnzbd remains a version behind.  am i doing something wrong?

**Comment by tteck** [comment]
To do

**Comment by tteck** [comment, marked as answer]
Ok, you can give it a try. Make sure you back it up first! Please let me know how it goes.

**Reply by ApriliaEdd** [reply]
Container updated correctly and appears to be working fine afterwards.  thanks!
